cabby/geo/map_processing/map_structure.py

```diff
@@ -240,17 +246,16 @@ def load_map(self, dir_name: Text):
     assert os.path.exists(
       path), "Path {0} doesn't exist.".format(path)
     poi_pandas = pd.read_pickle(path)
-    poi_pandas['cellids'] = poi_pandas['cellids'].apply(
-      lambda x: util.s2cells_from_cellids(x))
+    if 'cellids' in poi_pandas:
+      poi_pandas['cellids'] = poi_pandas['cellids'].apply(
+        lambda x: util.s2cells_from_cellids(x))
```

Comment:
Why not have a column for 's2cell' instead of overwriting what was in 'cellids'?

Reply:
cellids is s2cells. Its just that when I write to file I can't write the s2cells so I do: 
    pd_streets['cellids'] = pd_streets['cellids'].apply(
      lambda x: util.s2ids_from_s2cells(x))
and when I load I do:
      poi_pandas['cellids'] = poi_pandas['cellids'].apply(
        lambda x: util.s2cells_from_cellids(x))

Reply:
This is still confusing. Can you add some comments, including a bit of explanation and an example to illustrate what is happening?

Reply:
changed